presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -719,6 +719,19 @@ public Expression rewriteSymbolReference(SymbolReference node, Void context, Exp
             return builder.build();
         }
 
+        private List<VariableReferenceExpression> canonicalizeAndDistinctVariable(List<VariableReferenceExpression> outputs)
+        {
+            Set<VariableReferenceExpression> added = new HashSet<>();
+            ImmutableList.Builder<VariableReferenceExpression> builder = ImmutableList.builder();
```

Comment:
not need to have the builder; just use `ImmutableSet.copyOf(added)` for return statement

Reply:
It's the same logic as the symbol version. As a refactoring it's probably better to not introduce unnecessary changes.